decidim-core/vendor/assets/javascripts/form_datepicker.js.es6

```diff
@@ -20,7 +20,7 @@
         leftArrow: '<<',
         rightArrow: '>>'
       }).on('changeDate', (ev) => {
-        $(ev.target).siblings('input').val(exports.moment.utc(ev.date));
+        $(ev.target).siblings('input').val(exports.moment(ev.date));
```

Comment:
@oriolgual Which one is the correct timezone? Shouldn't decidim work on apps with arbitrary timezones? In any case, I didn't touch anything, this is just our standard sample app. The only difference is my local timezone, I think.